hadoop-hdds/docs/content/design/storage-distribution.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Storage Capacity Distribution Dashboard
+summary: Proposal for introducing a comprehensive storage distribution dashboard in Recon for enhanced cluster monitoring and debugging capabilities.
+date: 2025-08-05
+jira: HDDS-13177
+status: Under Review
+author: Priyesh Karatha
+---
+
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+  either express or implied. See the License for the specific
+  language governing permissions and limitations under the License.
+-->
+
+# Abstract
+
+Ozone currently lacks a unified interface to monitor and analyze storage distribution across its cluster components. This makes it difficult to:
+
+- Understand data distribution across the cluster
+- Debug storage reclamation issues
+- Monitor pending deletion progress
+- Analyze utilization patterns
+- Identify potential bottlenecks and imbalances
+
+This proposal introduces a comprehensive **Storage Capacity Distribution Dashboard** in Recon to address these challenges.
+
+---
+
+# Key Features
+
+## 1. Storage Distribution Analysis
+
+Detailed breakdown of storage usage across components:
+
+- **Global Used Space**
+- **Global Namespace Space**
+- **Open Keys and Open Files**: Data held in open keys and files
+- **Committed Keys**: Space used by committed key-value pairs
+- **Component-wise Distribution**: Metrics segregated by OM, SCM, and DataNodes
```

Comment:
These need more concrete definitions. What is the difference between global namespace and used space? Is this referring to replicated and un-replicated size? Then what is the difference between global namespace and committed keys? What does "storage usage" mean for OM and SCM? Is this the size of their DBs and Ratis logs, or is this referring to the size of block and containers their metadata is referencing?